test-suite/output/bug_12058.out

```diff
@@ -0,0 +1,11 @@
+Module
+FooBar
+:= Struct Definition foo : forall A : Type@{u}, A -> A.
```

Comment:
This should probably be 
```suggestion
:= Struct Definition foo@{u} : forall A : Type@{u}, A -> A.
```
so I wouldn't say the bug is fully fixed.

Reply:
Shall I close this PR and update the issue then?